pages/test_analytics/javascript_collectors.md

```diff
@@ -121,6 +122,30 @@ To configure Mocha:
     }
     ```
 
+### Cypress
+To configure Cypress:
+
+1. Make sure Cypress runs with access to [CI environment variables](/docs/test-analytics/ci-environments).
+1. Update your [Cypress Configuration](https://docs.cypress.io/guides/references/configuration)
+
+    ```js
+    // cypress.config.js
+
+    // Send results to Test Analytics
+    reporter: "buildkite-test-collector/cypress/reporter",
+    ```
+
+1. If you would like to pass in the API token using a custom environment variable, you can do so using the `reporterOptions`.
+
+    ```js
+    // cypress.config.js
+
+    // Send results to Test Analytics
+    reporterOptions: {
+      token_name: "CUSTOM_ENV_VAR_NAME"
+    }
+    ```
```

Comment:
Since this is not a separate step, but is part of step 2, I'd make this a [callout that's part of a list](https://github.com/buildkite/docs/blob/main/styleguides/markdown-syntax-style.md#callouts-within-lists).

```suggestion
    **Note:** To pass in the API token using a custom environment variable, add the `reporterOptions` option to your Cypress configuration:

    ```js
    // cypress.config.js

    // Send results to Test Analytics
    reporterOptions: {
      token_name: "CUSTOM_ENV_VAR_NAME"
    }
    ```
```

Reply:
Oh! This is so much better ✨